test/examples/break1.rb

```diff
@@ -4,6 +4,10 @@
 # A toy example for testing break commands.
 #
 class Break1Example
+  undef a if method_defined? :a
+  undef b if method_defined? :b
+  undef c! if method_defined? :c!
+
```

Comment:
Oh, in the end I had to do this another way, since using just `Break1Example.new.a` in the test would make the line-based breakpoints be set up inside `breakpoints_test.rb` instead of `break1.rb`.